replica/distributed_loader.cc

```diff
@@ -626,6 +626,10 @@ class table_population_metadata {
         return _highest_generation;
     }
 
+    using allow_offstrategy_compaction = bool_class<struct allow_offstrategy_compaction_tag>;
+    using must_exist = bool_class<struct must_exist_tag>;
+    future<> populate_column_family(sstring subdir, allow_offstrategy_compaction, must_exist = must_exist::yes);
+
```

Comment:
please rename class to table_sstable_loader / table_populator (table_population_metadata was just providing metadata, not actually performing any action), and populate_column_family can be renamed to just populate().